t/43-cli-api.t

```diff
@@ -495,4 +498,31 @@ subtest 'PIPE input' => sub {
     is $data->{body}, 'Hello openQA!', 'request body';
 };
 
+subtest 'YAML Templates' => sub {
+    my ($stdout, $stderr, @result) = capture sub { $api->run(@host, '-X', 'POST', 'job_groups', 'name=Test'); };
+    warn "stdout=[$stdout], stderr[$stderr], results[@result]";
```

Comment:
Please remove the unconditional warnings in this subtest as well.

Reply:
Same reason as above. Sorry for the noise.